Aerospace Investment.md

```diff
@@ -0,0 +1,64 @@
+# Summary
+This is a two-person scored negotiation simulation involving a venture capital investment. Individuals are scored on their ability to attain favorable investment terms for themselves and on the quality of the relationship they develop with their potential business partner. The simulation introduces the incorporation of process and relationship interests into negotiation strategy. Relationship-straining conflict has been purposefully included in the negotiation to assess students’ ability to deal with difficult demands while maintaining a positive relationship.
+
+
+# References
+
+High level overview: https://www.pon.harvard.edu/shop/aerospace-investment/
+
+Watermarked instructors copy including instructions, learning objectives, and scoring metrics: https://www.pon.harvard.edu/wp-content/uploads/images/posts/Aerospace_Teachers_Package_Watermarked-1.pdf
+
+
+# Procedure
+## Steps
+1. Participants read the instructions
```

Comment:
should this include the general instructions and confidential instructions?

Reply:
Should 'prep' time be included somewhere in the procedure too? Since people need to prepare for the negotiations?

Reply:
Prep time is a good point! I'll add that. As for the instructions, I was going to add them but they are about five pages long and there is no way to copy/paste them because the pdf is watermarked. I can add them but wanted some feedback first.